interface/src/index.html

```diff
@@ -15,12 +15,29 @@
     <!-- font awesome from BootstrapCDN -->
     <link href="http://netdna.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.css" rel="stylesheet">
 
+
+    <!-- Required CSS for ui-slider -->
+    <link rel="stylesheet"
+          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+
     <!-- compiled CSS --><% styles.forEach( function ( file ) { %>
     <link rel="stylesheet" type="text/css" href="/static/<%= file %>" /><% }); %>
 
+    <!-- Required js for ui-slider
+         FIXME: Why adding these two libraries in build.config.js does not work?
+        -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js">
+    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js">
+    </script>
+
     <!-- compiled JavaScript --><% scripts.forEach( function ( file ) { %>
     <script type="text/javascript" src="/static/<%= file %>"></script><% }); %>
 
+    <!-- d3.network library, written by Aaron Wang (aaronkw) -->
+    <script src="https://rawgit.com/dongbohu/d3-network/master/d3.network.js"
+            charset="utf-8"></script>
```

Comment:
Because **d3.network** in active development now, so I didn't include it in **build.config.js**. If I did, every time d3.network changes, I will have to copy it to **vendor/** directory.